R/GTM.R

```diff
@@ -0,0 +1,66 @@
+#' GTM
+#' This function carries out gene text mining.
+#' @param genes A character string containing the genes
+#' to perform text mining using PubMed database
+#' @param query_string A character string containing words 
+#' in query to follow the gene of interest. Default is 
+#' "AND cancer AND driver" resulting in a final query of
+#' "Gene AND cancer AND driver". Standard PubMed syntax
+#' can be used in the query. For example Boolean operators
+#' AND, OR, NOT can be applied and tags such as [AU], 
+#' [TITLE/ABSTRACT], [Affiliation] can be used. 
+#' @import easyPubMed
+#' @import tibble
+#' @importFrom magrittr "%>%"
+#' @importFrom purrr map
+#' @importFrom dplyr rename
+#' @return A tibble containing results of text mining where
+#' PubMed was queried for information of input genes. Each
+#' row in the tibble contains a PubMed ID for a given gene.
+#' If more than 20 results from the specified query is obtained, 
+#' only the first 20 are included. The tibble contains four
```

Comment:
I think it would be good to have an option to specify the number of results - just to ensure maximum flexibility (unless we have a strong reason for which they should be 20)

Reply:
I've added a function argument called max_records that supports this